### ISSUE TITLE
Add unit tests for rpc/v1/subscribe.go

### ISSUE THREAD

**Issue by matheusmosca** [matheusmosca/walrus#32]
You can find the file [here](https://github.com/matheusmosca/walrus/blob/main/rpc/v1/subscribe.go). Take a look at [this StackOverflow post](https://stackoverflow.com/questions/42102496/testing-a-grpc-service) for more info about gRPC testing in Go.

**Comment by Smelton01** [comment]
Hi, I'm fairly new to Go but I would like to help with this. Can you please assign it to me.

**Comment by matheusmosca** [comment]
Hi @Smelton01, I would suggest #31 since it is a simpler use case. Which one do you prefer?

**Comment by Smelton01** [comment]
@matheusmosca yeah I can try and write the tests for #31 first. Sorry but can you reassign me to that one instead.

**Comment by matheusmosca** [comment]
Could you send a comment on there? I'm not able to assign an issue to someone who isn't participating in the issue
